Sources/Crypto/Key Derivation/ANSIx963.swift

```diff
@@ -18,8 +18,12 @@
 #if CRYPTOKIT_NO_ACCESS_TO_FOUNDATION
```

Comment:
Just checking: does `CRYPTOKIT_NO_ACCESS_TO_FOUNDATION` imply no access to `FoundationEssentials` either?